test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -33,7 +33,7 @@
     @Before
     public void setUp() throws Exception {
         Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
-                new Email("john@doe.com", false), new Address("395C Ben Road", false), new UniqueTagList());
+                new Email("john@doe.com", false), new Address("395G, Ben Road, #12-23, 123445", false), new UniqueTagList());
```

Comment:
Good work on updating the unit tests. However, remember you should also add tests cases in the system level tests (i.e. the input.txt + actual.txt)